distribution/ddf-common/src/main/resources/etc/artemis.xml

```diff
@@ -27,6 +27,9 @@
                 </anycast>
             </address>
         </addresses>
+        <broker-plugins>
+            <broker-plugin class-name="org.codice.ddf.broker.security.BrokerAuthenticationPlugin" />
```

Comment:
do these same changes need to be made to artemis-backup.xml?

Reply:
fixed